app/assets/javascripts/application.js

```diff
@@ -16,3 +16,7 @@
 //= require jquery-ui
 //= require turbolinks
 //= require_tree .
+//= require js-routes
+
+window.pusher = new Pusher(window.pusher_key);
```

Comment:
Hmm, not sure if I got that error. I was storing the key in the environment variables (.env on my local computer) so maybe that's why. But I'm actually not sure that the key needs to be secret (it's in the default script you're supposed to include -- and there's another explicitly secret key) so I'll just hardcore it here as well.